projects/core/src/model/cms.model.ts

```diff
@@ -204,6 +204,10 @@ export interface CmsOrderDetailItemsComponent extends CmsComponent {
   enableAddToCart?: boolean;
 }
 
+export interface CmsOrderDetailReorderComponent extends CmsComponent {
+  enableReorder?: boolean;
```

Comment:
`enableReorder` does not seem to be used anywhere. Is it the case? If so, perhaps `CmsOrderDetailReorderComponent` is not required?